vtex/omnishipping
# Fix not saving selected pickup point while switching delivery channels

#### What is the purpose of this pull request?
Ditto.

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/27173/investiga%C3%A7%C3%A3o-alternar-entre-deliverychannel-perde-pickup-selecionado-pelo-usu%C3%A1rio)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1.myvtex.com/checkout/cart/add/?sku=298&qty=1&seller=1&sc=1)
2. Proceed to Shipping and type postal code 22250-040
3. Select Pickup and it will pre-select `Copacabana`
4. Open the Pickup points modal and select `Flamengo`
3. Select Delivery than Pickup again and `Flamengo` should remain selected

#### Screenshots or example usage
n/a
#### Types of changes
